com.unity.shadergraph/Editor/Data/Nodes/UV/FlipbookNode.cs

```diff
@@ -143,17 +143,7 @@ public void GenerateNodeFunction(FunctionRegistry registry, GenerationMode gener
                     FindOutputSlot<MaterialSlot>(OutputSlotId).concreteValueType.ToShaderString());
                 using (s.BlockScope())
                 {
-                    string offset;
-                    if (concretePrecision == ConcretePrecision.Half)
-                    {
-                        offset = "0.00001h";
-                    }
-                    else
-                    {
-                        offset = "0.000001";
-                    }
-
-                    s.AppendLine("Tile = floor(fmod(Tile + " + offset + ", Width*Height));");
+                    s.AppendLine("Tile = floor(fmod(Tile + $precision(0.00001), Width*Height));");
```

Comment:
This is equivalent and passes validation (you're not allowed to change templated function code based on concrete precision -- you must use the precision token to do that)